onboarding-website/src/components/asideComponent.vue

```diff
@@ -0,0 +1,73 @@
+<!-- eslint-disable vue/no-v-text-v-html-on-component -->
+<template>
+  <v-card class=" search-bar mx-auto" >
+    <v-toolbar color="primary" >
+      <v-text-field
+        v-model="data.search"
+        clearable
+        hide-details
+        label="Search Topics"
+        single-line />
+        <template v-slot:prepend>
+        <v-avatar class="me-4 mt-2" rounded="0">
+        <img src="/src/assets/magnify.svg" alt="" srcset="">
+        </v-avatar>
+        </template>
+    </v-toolbar>
+
+  
+
+    <v-list>
+      <v-list-item v-for="(item, i) in searching" :key="i" link>
```

Comment:
What is `i`? In quantitative programming, we usually have a practice to use `i` as a variable, whenever you write any code while developing any application use a quality name and descriptive too.

Reply:
resolve